python-lib/ontology_tagging/formatting/formatter_by_match.py

```diff
@@ -103,21 +103,20 @@ def _get_tags_in_row(self, matches: List, row: pd.Series, language: AnyStr) -> N
         Called by _write_row
         Create new rows from the input one, containing the tag, keyword, and sentence columns for each match, and append them to the output_df.
         """
-        values = []
+        tag_rows = []
```

Comment:
This is still not needed 😛  : You both define and use `tag_rows` at each iteration of the for loop below, so this empty list value will never be used